equinix/resource_metal_device.go

```diff
@@ -9,10 +9,10 @@ import (
 	"path"
 	"reflect"
 	"regexp"
+	"slices"
 	"sort"
 	"strings"
 	"time"
-
```

Comment:
This commit, "refactor: replace isStringInSlice with slices.Contains" could be merged with the previous commit. There are only minor fmt'ing changes introduced.